dissect/evidence/tools/asdf/dd.py

```diff
@@ -45,7 +47,7 @@ def fill_zero(fhout: BinaryIO, length: int) -> None:
     fhout.write(b"\x00" * remain)
 
 
-def main():
+def main() -> None:
```

Comment:
Don't you want to return a status code here? If so, this could be done in a separate PR when necessary.